src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -301,8 +301,8 @@ private int execMatlabCommand(FilePath workspace, Launcher launcher,
     public String constructCommandForTest(FilePath scriptPath) {
         final String matlabScriptName = getValidMatlabFileName(scriptPath.getBaseName());
         final String runCommand = "addpath('" + scriptPath.getRemote().replaceAll("'", "''")
-                + "'); " + matlabScriptName + ",delete('.matlab/" + scriptPath.getBaseName() + "/"
-                + matlabScriptName + ".m'),runnerScript,rmdir(tmpDir,'s')";
+                + "');tmpDir=tempname;mkdir(tmpDir), ,movefile('.matlab/" + scriptPath.getBaseName() + "/"

```

Comment:
Why move file here, Cant we keep the runner script as is and just call it here with simple script name

Reply:
@dolonm-mw can you pls look into this

Reply:
On it!